cmd/rekor-server/app/serve.go

```diff
@@ -49,12 +50,36 @@ var serveCmd = &cobra.Command{
 		}
 		slog.Info("starting rekor-server", "version", versionInfoStr)
 
+		readOnly := viper.GetBool("read-only")
+		persistentAntispam := viper.GetBool("persistent-antispam")
+		asMaxBatchSize := viper.GetUint("antispam-max-batch-size")
+		asPushbackThreshold := viper.GetUint("antispam-pushback-threshold")
+
+		gcpBucket := viper.GetString("gcp-bucket")
+		gcpSpannerDb := viper.GetString("gcp-spanner")
+
 		// currently only the GCP driver is supported for rekor-tiles.
-		tesseraDriver, err := tessera.NewGCPDriver(ctx, viper.GetString("gcp-bucket"), viper.GetString("gcp-spanner"))
-		if err != nil {
-			slog.Error(fmt.Sprintf("failed to initialize GCP driver: %v", err.Error()))
+		var tesseraDriver tesseraLib.Driver
+		var antispamProvider tesseraLib.Antispam
+		switch {
+		case gcpBucket != "" && gcpSpannerDb != "":
+			tesseraDriver, err = tessera.NewGCPDriver(ctx, gcpBucket, gcpSpannerDb)
+			if err != nil {
+				slog.Error(fmt.Sprintf("failed to initialize GCP driver: %v", err.Error()))
+				os.Exit(1)
+			}
+			if !readOnly && persistentAntispam {
+				antispamProvider, err = tessera.NewGCPAntispam(ctx, gcpSpannerDb, asMaxBatchSize, asPushbackThreshold)
```

Comment:
is there anything to indicate that antispam was correctly configured/initialized. As an optional configuration parameter, should we make it more explicit that it is enabled so it shows up in server initialization logs?

Reply:
`NewDriver` will fail if the persistent antispam provider fails to be initialized. Otherwise, there is no way for non-persistent antispam to err out on startup.